packages/flutter/lib/src/material/stepper.dart

```diff
@@ -938,3 +999,171 @@ class _TrianglePainter extends CustomPainter {
     );
   }
 }
+
+/// This class is used to override the default visual properties of [Step] widgets within a [Stepper].
+///
+/// To customize the appearance of a [Step] create an instance of this class with non-null parameters
+/// for the step properties whose default value you want to override.
+///
+/// Example usage:
+/// ```dart
+/// Step(
+///   title: const Text('Step 1'),
+///   content: const Text('Content for Step 1'),
+///   stepStyle: StepStyle(
+///     color: Colors.blue,
+///     errorColor: Colors.red,
+///     border: Border.all(color: Colors.grey),
+///     boxShadow: const BoxShadow(blurRadius: 3.0, color: Colors.black26),
+///     gradient: const LinearGradient(colors: <Color>[Colors.red, Colors.blue]),
+///     indexStyle: const TextStyle(color: Colors.white),
+///   ),
+/// )
+/// ```
+/// See also:
```

Comment:
We typically keep "See also:" for links to other topics, not for examples.  It will render strangely on the API docs page anyhow, since the example will be in a separate box from the "See also:" text.

```suggestion
///
```